### DISCUSSION TITLE
Collaborator Logo Collage

### DISCUSSION THREAD

**Discussion by Gene-Weaver** [Gene-Weaver/VoucherVision#12] [General Info & Subgroups]
If you don't see your logo or want me to use a different logo, let me know!

[![Partners](https://LeafMachine.org/partners/collage.jpg "Partners")](https://docs.google.com/forms/d/e/1FAIpQLSe2E9zU1bPJ1BW4PMakEQFsRmLbQ0WTBI2UXHIMEFm4WbnAVw/viewform?usp=sf_link)

**Comment by nickynicolson** [comment]
Since the [DarwinCore](https://dwc.tdwg.org/) standard is a core part of Voucher Vision, have you any thoughts on adding the [Biodiversity Information Standards (TDWG)](https://www.tdwg.org/) logo?  Resources available here: https://www.tdwg.org/about/logos/
